File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        var ans = new StringBuilder();
        while(columnNumber > 0){
            columnNumber--;
            var remainder = columnNumber % 26;
            columnNumber /= 26;

            var ch = (char) ('A' + remainder);
            ans.append(ch);
        }

        return ans.reverse().toString();
    }
}

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        ans = ""
        while columnNumber > 0:
            columnNumber -= 1
            remainder = columnNumber % 26
            columnNumber = columnNumber // 26

            ch = chr(ord("A") + remainder)
            ans += ch

        return ans[::-1]